libc/config/linux/app.cpp

```diff
@@ -0,0 +1,13 @@
+//===-- Implementation file for linux applications classes  -----*- C++ -*-===//
```

Comment:
why was this file added?

Reply:
`AppProperties app` was defined in `start.cpp`; I moved its definition here.